src/main/pg/sdcard.c

```diff
@@ -56,6 +56,10 @@ void pgResetFn_sdcardConfig(sdcardConfig_t *config)
     config->mode = SDCARD_MODE_NONE;
 #endif
 
+#if defined(STM32H7) && defined(USE_SDCARD_SDIO) // H7 only for now, likely should be applied to F4/F7 too
```

Comment:
This needs to be removed, see https://github.com/betaflight/betaflight/pull/10705#discussion_r644453878.

Also, looking at the code in https://github.com/betaflight/betaflight/pull/10573/files#diff-a50a03208d239013f47765a2b52d22cb257bbf93fd578ec628b1be73ede726d0R48-R57, this should be already taking care of making it possible to load the config for targets that have it on SD card.

Reply:
Apologies. Somehow missed that one. Removed.